### ISSUE TITLE
Bootstrap dependency

### ISSUE THREAD

**Issue by redmagic** [formio/angular-material-formio#156]
Hi,

Maybe it would be good to remove the ngx-bootstrap dependency? As material brings it's own styliing, we could get rid of bootstrap. It adds extra weight and ngcc compile time as it is now.

**Comment by travist** [comment]
Agreed. We will remove it when we accept the Angular 10 upgrade pull request.

**Comment by abdulfousan** [comment]
@travist Could u plz share any updates on this?

**Comment by travist** [comment]
No updates, but we do need an effort to upgrade this library with latest @formio/angular@5.x versions since this works with latest Angular. We do not have any active projects right now on this library so we will either need to wait for that, or maybe the community could lend a hand and submit some pull requests that update this library?
